### ISSUE TITLE
Dream booth unidentified image !??

### ISSUE THREAD

**Issue by iano0o** [googlecolab/colabtools#3252]
am training using dream booth google collab  
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb#scrollTo=O3KHGKqyeJp9

when i get to the point where i need to upload the images this error appear after i upload .... i did this yesterday it did not happen !!

![Capture](https://user-images.githubusercontent.com/119347735/204346945-563ee150-bb00-4a02-8cce-168216a2e19d.PNG)

**Comment by mco-gh** [comment]
- Have you tried restarting the kernel and running the entire notebook, top to bottom? Unexpected things can happen if you rerun a cell in a notebook in an unknown state.
- If the above is not the issue, have you tried checking with the party that provided the notebook?

**Comment by iano0o** [comment]
> * Have you tried restarting the kernel and running the entire notebook, top to bottom? Unexpected things can happen if you rerun a cell in a notebook in an unknown state.
> * If the above is not the issue, have you tried checking with the party that provided the notebook?

i have sloved the problem !!!

so all you need to do is

add new cell ....add this line !pip install pillow==8.4 
then run the cell then restart the whole runtime then it will work!!

**Comment by mayankmalik-colab** [comment]
I just did some digging in the github repo for that notebook and realized that there have been many changes with dependencies. In the recent commits, a few `!pip install` commands have been removed from the notebook, especially  for `pillow` package. Therefore, these changes affected the notebook executions. Most of the times the best bet is to check the notebook's github repo or connect with the authors directly. I am glad it worked for you eventually.
